packages/components/src/pages/Quickstart.vue

```diff
@@ -0,0 +1,775 @@
+      </div>
+      <section class="qs-step" v-if="currentStep === 1">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Install AppMap Agent</h1>
+          <select class="qs-select" v-model="selectedLanguage">
+            <option
+              v-for="lang in Object.keys(languages)"
+              :key="lang"
+              :value="lang"
+            >
+              {{ languages[lang] }}
+            </option>
+          </select>
+        </div>
+        <div class="qs-step__block" v-if="!step1Completed && !error">
+          <p>
+            This will add the following snippet to the top of your Gemfile and
+            run bundle to install the AppMap gem.
+          </p>
+          <code class="qs-code" @click="select">{{
+            installSnippets[selectedLanguage]
+          }}</code>
+          <button class="qs-button" v-if="!isActionRunning" @click="runAction">
+            Install the AppMap agent
+          </button>
+          <div class="qs-loader" v-if="isActionRunning">
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+            <div class="qs-loader__dot"></div>
+          </div>
+        </div>
+        <div class="qs-step__error" v-if="error">
+          <span class="qs-step__error-title">{{ error }}</span>
+          <ol class="qs-step__error-list">
+            <li class="qs-step__error-list-item">Review the console output</li>
+            <li class="qs-step__error-list-item">Make necessary adjustments</li>
+            <li class="qs-step__error-list-item">
+              <a href="#">Try again to install</a>
+            </li>
+          </ol>
+          <a
+            class="qs-step__error-button qs-button qs-button--bordered"
+            href="https://appland.com/company/contact-us"
+            target="_blank"
+            rel="noopener noreferrer"
+            >For further assistance please contact us</a
+          >
+        </div>
+        <div class="qs-step__success" v-if="step1Completed && !error">
+          <span class="qs-step__success-title">
+            <SuccessIcon class="qs-step__success-icon" />
+            Agent installed
+          </span>
+          <button
+            type="button"
+            class="qs-step__success-next-step qs-button"
+            @click="nextStep"
+          >
+            Next step : Configure AppMap ->
+          </button>
+        </div>
+      </section>
+      <section class="qs-step" v-if="currentStep === 2">
+        <div class="qs-step__head">
+          <h1 class="qs-title">Configure AppMap</h1>
+        </div>
+        <div class="qs-step__block" v-if="!step2Completed">
+          <p>
+            This will create an <a href="#">appmap.yml</a> config file in the
+            root directory of your project. This will tell AppMap what code to
+            record. You can run these defaults or add more packages, gems, or
+            specific functions. You can edit this file at any time.
```

Comment:
This text will also need to be changed depending on the language selected, or the text needs to be made more generic @ptrdvrk

Reply:
Apparently, new languages won't be added often.
Maybe we will write texts for every existing language and swap them inside Quickstart.vue when the user change language in select?

Reply:
I'm fine with that for now.